frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,22 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size…")}
```

Comment:
Since we know whether the font size is being increased or decreased it might be better to be specific?